openstack/swift/aggregations/swift-usage.rules

```diff
@@ -1,10 +1,10 @@
 groups:
 - name: swift-usage
   rules:
-  - record: swift_cluster_storage_used_percent_average
+  - record: global:swift_cluster_storage_used_percent_average
     expr: avg(swift_cluster_prostorage_used_percent_gauge_average < inf)
 
-  - record: swift_cluster_storage_used_percent_gauge_average
+  - record: global:swift_cluster_storage_used_percent_gauge_average
```

Comment:
Maybe that's nicer: If you prefix a metric with `global:` it automatically gets federated to the global prometheus and no one needs to maintain the ugly list.